src/sidebar/store/modules/annotations.js

```diff
@@ -192,19 +201,28 @@ const reducers = {
     return { annotations: anns };
   },
 
-  /** @param {State} state */
+  /**
+   * @param {State} state
+   * @param {{ highlighted: Record<string, boolean> }} action
+   */
   HIGHLIGHT_ANNOTATIONS(state, action) {
     return { highlighted: action.highlighted };
   },
 
-  /** @param {State} state */
+  /**
+   * @param {State} state
+   * @param {{ annotationsToRemove: AnnotationStub[], remainingAnnotations: Annotation[] }} action
```

Comment:
The `remainingAnnotations` field looks like a code smell to me, but in this PR I'm just documenting/typing the existing code.